sdks/python/apache_beam/transforms/display.py

```diff
@@ -141,7 +147,7 @@ def create_from_options(cls, pipeline_options):
 
     items = {k: (v if DisplayDataItem._get_value_type(v) is not None
                  else str(v))
-             for k, v in pipeline_options.display_data().items()}
+             for k, v in iteritems(pipeline_options.display_data())}
```

Comment:
Does this actually need to change?

Reply:
I added for efficiency in Python2, but this will actually not be called that often, so I'll revert it.